houston/deployment.go

```diff
@@ -347,6 +347,9 @@ var (
 						type
 						url
 					}
+					dagDeployment {
```

Comment:
just checking if this new field is present in all Houston versions from 0.29 onwards or added recently.

Reply:
Yes it is